justfile

```diff
@@ -124,8 +124,11 @@ logs-cloudbeat:
 deploy-cloudformation:
   cd deploy/cloudformation && go run .
 
+deploy-dm-ci:
+  cd deploy/deployment-manager; . ./set_env.sh && ./deploy.sh
+
 deploy-dm:
- ./deploy/deployment-manager/deploy.sh
+ cd deploy/deployment-manager; ./deploy.sh
```

Comment:
why change the directory instead of calling the full path of the script? this could lead to unexpected issues

Reply:
The `deploy.sh` script references a template file named `compute_engine.py`. This template file is expected to be in the same directory as the script. If you run `deploy.sh` without changing to the script's directory first, you'll encounter a 'file not found' error.
```
# Apply the deployment manager templates
run_command "gcloud deployment-manager deployments create --automatic-rollback-on-error ${DEPLOYMENT_NAME} --project ${PROJECT_NAME} \
    --template compute_engine.py \
    --properties elasticAgentVersion:${STACK_VERSION},fleetUrl:${FLEET_URL},enrollmentToken:${ENROLLMENT_TOKEN},allowSSH:${ALLOW_SSH},zone:${ZONE},elasticArtifactServer:${ELASTIC_ARTIFACT_SERVER}"

```

Reply:
the code has been reverted